examples/default_level.rs

```diff
@@ -0,0 +1,33 @@
+use clap::Parser;
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+
+/// Foo
+#[derive(Debug, Parser)]
+struct DefaultCli {
+    #[command(flatten)]
+    verbose: Verbosity,
+}
+
+/// Bar
```

Comment:
Whats with the doc comments?

Reply:
just using what was in the other example and expanding on it, I will remove them